vnext/Desktop.UnitTests/WinRTWebSocketResourceUnitTest.cpp

```diff
@@ -62,10 +62,8 @@ TEST_CLASS (WinRTWebSocketResourceUnitTest) {
     Assert::IsTrue(connected);
   }
 
-  // TODO: Re-enable. Fails in x86|Release.
-  //      Hangs due to exception not being caught within WinRTWebSocketResource::PerformConnect.
   BEGIN_TEST_METHOD_ATTRIBUTE(ConnectFails)
-  TEST_IGNORE()
+  TEST_IGNORE() // TODO: Remove before merging. Ensure mocks behave as original APIs.
```

Comment:
Mocks still throw and the adapter doesn't translate the behavior to HRESULT.
Fix before merging.